utilities/audio.py

```diff
@@ -1,8 +1,8 @@
 import moviepy.editor as mp
 
 def convert_to_wav(path):
-    output_path = "/voice.wav"
+    output_path = "/home/dk/Downloads/out-put-voice.wav"
```

Comment:
just for Permission issues, this needed to be changed !

Reply:
This one may not work, since this path does not exist in every computer

Reply:
Yes, it was for my personnal testing !
That's why i said it was for "permission issues"
You can get rid of this change !